src/components/molecules/Wallet/Details.tsx

```diff
@@ -75,7 +111,18 @@ export default function Details(): ReactElement {
                 onChange={handlePortisNetworkChange}
               />
             )} */}
-          </span>
+            {providerInfo?.name === 'MetaMask' && (
+              <Button
+                style="text"
+                size="small"
+                onClick={() => {
+                  addOceanToWallet()
+                }}
+              >
+                Add Ocean
```

Comment:
aaaand right, another symbol :-) We could use same check as above, but shouldn't be any blockers to getting that PR on ocean.js released quickly, so we can make use of `oceanTokenSymbol`. Only the CI over there is slow

Reply:
pr+release at best 30-40 minutes

Reply:
right, I'll change that after release. I see that with the changes from main the tokenSymbol from wallet `balance` is also changed to `mOCEAN`. Didn't changed what you spotted to keep it consistent to what the value was before changes.

Reply:
yup, those symbol hacks come in from multiple PRs now. If anybody can approve my PR I can handle the release https://github.com/oceanprotocol/ocean.js/pull/670